src/languages/demandwarescript.coffee

```diff
@@ -0,0 +1,115 @@
+# Get Atom defaults
+scope = ['source.js']
+tabLength = atom?.config.get('editor.tabLength', scope: scope) ? 4
+softTabs = atom?.config.get('editor.softTabs', scope: scope) ? true
+defaultIndentSize = (if softTabs then tabLength else 1)
+defaultIndentChar = (if softTabs then " " else "\t")
+defaultIndentWithTabs = not softTabs
+
+module.exports = {
+
+  name: "Demandware Script"
+  namespace: "ds"
+
+  ###
+  Supported Grammars
+  ###
+  grammars: [
+    "Demandware Script"
+  ]
+
+  ###
+  Supported extensions
+  ###
+  extensions: [
+    "ds"
+  ]
+
+  defaultBeautifier: "DS Beautify"
+
+  ###
+
+  ###
+  options:
+    # JavaScript
```

Comment:
Is this simply `JavaScript` language? If so, then use `fallback: ['js']` instead of copying all of the options and duplicating them 👍